src/components/IdentityRadioList.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <AeList class="list">
+    <ListItem
+      :subtitle="totalBalance + ' AE'"
+      :type="'default'"
+    >
+      <div
+        slot="icon"
+        class="avatar"
+      >
+        ↪
+      </div>
+      Total Balance
+    </ListItem>
+
+    <ListItem
+      v-for="(identity, index) in identities"
+      :id="index"
+      :subtitle="identity.balance+' AE'"
+      :checked="index === selectedIdentityIdx"
+      :key="index"
+      :type="'radio'"
+      @change="selectIdentity(index)"
+    >
+      <ae-identity-avatar
+        slot="icon"
+        :address="identity.address"
+        class="avatar"
+      />
+      {{ identity.name }}
+    </ListItem>
+  </AeList>
+</template>
+
+<script>
+import ListItem from './ListItem'
+import { mapState, mapGetters, mapMutations } from 'vuex'
+import { AeIdentityAvatar } from '@aeternity/aepp-components'
+import AeList from '../components/AeList'
+
+export default {
+  name: 'IdentityRadioList',
+  components: { ListItem, AeIdentityAvatar, AeList },
+  computed: {
+    ...mapGetters(['identities', 'totalBalance']),
```

Comment:
Probably some of this variables are unused in Accounts page anymore, don't forget to remove them from there.